include/revng/Pipes/PipelineManager.h

```diff
@@ -83,6 +83,9 @@ class PipelineManager {
   /// Path
   llvm::Error overrideModel(llvm::StringRef Path);
 
+  /// Entirelly replaces the model with the one provided
```

Comment:
* Why do we still have model-specific methods in `PipelineManager`? I was hoping it could decay to be a super-regular global variable after the recent changes.
* The other method should be called `overrideModelFromFile`
* Please, pretty please, install a vim plugin spell checking comments and possibly also commit messages.
Pretty pretty please.

Reply:
it was mostly to implement the -m option in tools, i can move it to main if you prefer

Reply:
Yes, makes sense.

Reply:
toh multiple tools need to use it to implement -m so that is a better place to keep it.

Reply:
So you think there's a demand for an handy wrapper, gotcha.
Still he's a bit of special kid. Can we make this method non `extern` and make it a wrapper around an `overrideGlobal` method? Something like:
```c++
llvm::Error overrideModel(TupleTree<model::Binary> Binary) {
  return overrideGlobal<ModelGlobal>(ModelGlobalName, ...);
}
```

Reply:
i will push it in the new CLToolOption class